_episodes/06-func.md

```diff
@@ -355,7 +365,8 @@ we can break the string across multiple lines:
 
 ~~~
 def offset_mean(data, target_mean_value):
-    '''Return a new array containing the original data with its mean offset to match the desired value.
+    '''Return a new array containing the original data with its mean offset 
+    to match the desired value.
```

Comment:
Same as above: 
Split so that the second line says `with its mean offset to match the desired value` and
then shift text to the right to align it with `Return a new array`